app/templates/events/event_list.html

```diff
@@ -86,10 +86,10 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
             <tr class="">
           {% endif %}
             {% if type != "bonner" and type != "studentLed" %}
-              {% if event.program%}
-                <td class="text-dark">{{event.program.programName}}</td>
-              {% else %}
+              {% if event.isAllVolunteerTraining %}
```

Comment:
We don't really need to have this condition there anymore, because it has a program name